packages/storage/src/service.ts

```diff
@@ -88,145 +179,50 @@ export class StorageService {
 
   /**
    * Stop running requests and prevent more from being created.
+   * @internal
    */
   deleteApp(): void {
-    this.deleted_ = true;
-    this.app_ = null;
-    this.requests_.forEach(request => request.cancel());
-    this.requests_.clear();
+    this._deleted = true;
+    this._requests.forEach(request => request.cancel());
+    this._requests.clear();
   }
 
   /**
    * Returns a new firebaseStorage.Reference object referencing this StorageService
    * at the given Location.
-   * @param loc The Location.
+   * @internal
+   * @param loc - The Location.
    * @return A firebaseStorage.Reference.
    */
   makeStorageReference(loc: Location): Reference {
```

Comment:
This is not part of the public API, so it can be removed.

Reply:
Done.